src/js/api/core-loader.js

```diff
@@ -17,7 +18,7 @@ export default function loadCoreBundle(model) {
 
 export function chunkLoadErrorHandler(/* error */) {
     // Webpack require.ensure error: "Loading chunk 3 failed"
-    throw new Error('Network error');
+    throw SetupError(Code.NETWORK_ERROR);
```

Comment:
You mentioned extending Error, but this is not a class.

Reply:
I have a separate PR for that https://github.com/jwplayer/jwplayer-errors/pull/4